Index: TeamCode/src/main/java/org/timecrafters/javaClass/spencer/SpencerFirstState.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.timecrafters.javaClass.spencer;\r\n\r\nimport org.cyberarm.engine.V2.CyberarmState;\r\nimport org.timecrafters.javaClass.samples.SampleRobot;\r\n\r\npublic class SpencerFirstState extends CyberarmState {\r\n\r\n    //here, you'll find some of your variables. you can add more as you need them.\r\n    private SampleRobot robot;\r\n\r\n    //This is the constructor. It lets other code bits run use the code you put here\r\n    public SpencerFirstState(SampleRobot robot) {\r\n        this.robot = robot;\r\n    }\r\n\r\n    //This is a method. methods are bits of code that can be run elsewhere.\r\n    //This one is set up to repeat every few milliseconds\r\n    @Override\r\n    public void exec() {\r\n\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/TeamCode/src/main/java/org/timecrafters/javaClass/spencer/SpencerFirstState.java b/TeamCode/src/main/java/org/timecrafters/javaClass/spencer/SpencerFirstState.java
--- a/TeamCode/src/main/java/org/timecrafters/javaClass/spencer/SpencerFirstState.java	(revision a456b9a0029643112cbcd2236d74ae2477fb7278)
+++ b/TeamCode/src/main/java/org/timecrafters/javaClass/spencer/SpencerFirstState.java	(date 1628040144672)
@@ -7,7 +7,8 @@
 
     //here, you'll find some of your variables. you can add more as you need them.
     private SampleRobot robot;
-
+    private double rightStick;
+    private double leftStick;
     //This is the constructor. It lets other code bits run use the code you put here
     public SpencerFirstState(SampleRobot robot) {
         this.robot = robot;
@@ -17,6 +18,12 @@
     //This one is set up to repeat every few milliseconds
     @Override
     public void exec() {
+    rightStick = engine.gamepad1.right_stick_y;
+    leftStick = engine.gamepad1.left_stick_y;
+    robot.driveFrontLeft.setPower(-leftStick);
+    robot.driveFrontRight.setPower(-rightStick);
+    robot.driveBackLeft.setPower(-rightStick);
+    robot.driveBackRight.setPower(-leftStick);
 
     }
 }
Index: .idea/compiler.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"CompilerConfiguration\">\r\n    <bytecodeTargetLevel target=\"1.8\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/compiler.xml b/.idea/compiler.xml
--- a/.idea/compiler.xml	(revision a456b9a0029643112cbcd2236d74ae2477fb7278)
+++ b/.idea/compiler.xml	(date 1628037945639)
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="1.8" />
+    <bytecodeTargetLevel target="11" />
   </component>
 </project>
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_1_8\" default=\"true\" project-jdk-name=\"1.8\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/build/classes\" />\r\n  </component>\r\n  <component name=\"ProjectType\">\r\n    <option name=\"id\" value=\"Android\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision a456b9a0029643112cbcd2236d74ae2477fb7278)
+++ b/.idea/misc.xml	(date 1628037945616)
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" project-jdk-name="11" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
